src/UI/Art/SpriteArtist/MapArtist.java

```diff
@@ -45,19 +45,9 @@ public MapArtist(GameInstance game, MapView view)
 
     baseMapImage = new BufferedImage(gameMap.mapWidth * tileSize, gameMap.mapHeight * tileSize, BufferedImage.TYPE_INT_RGB);
 
-    // This nice little hack will semi-randomly decide what color fog is used for this game.
-    // It switches every 10 seconds, so two games started 10 seconds apart will have different fog settings.
-    String strTime = Long.toString(System.currentTimeMillis());
-    if( !myGame.isFogEnabled() || (Integer.parseInt(new String() + (strTime.charAt(strTime.length() - 5))) % 2 == 0) )
-    {
-      FOG_COLOR = new Color(72, 72, 96, 200); // dark blue
-      HIGHLIGHT_COLOR = new Color(255, 255, 255, 160); // white
-    }
-    else
-    {
-      FOG_COLOR = new Color(255, 255, 255, 200); // white
-      HIGHLIGHT_COLOR = new Color(128, 128, 255, 112); // bluish
-    }
+    // Choose colors for fog and highlighted tiles.
+    FOG_COLOR = new Color(72, 72, 96, 200); // dark blue
+    HIGHLIGHT_COLOR = new Color(255, 255, 255, 160); // white
```

Comment:
Got a solid decision on that, did you?
Works for me.

Reply:
White fog looks too much like snow.

Reply:
Did you check rain against the blue?

Reply:
Yeah. It's also not great, but it's not as bad as the white/snow, and eventually rain will be animated; the current approach is just a placeholder.